CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## [1.8.0] - 2022-11-20
+
+- added the option to define a toleration within the kubernetes_deployment_v1 resource
```

Comment:
please also add tolerations to the other pod resource types.

Also please keep in mind, the resource type files are ordered the same way, so please put the tolerations into the same position in each of them, because it makes diffing the files easier to check for missing things.

Reply:
added the same configuration to the other pod resource types: afab593e47e0ceeaca7c922d16a0b65e6a078f93